src/tools/illink/test/Mono.Linker.Tests.Cases/DataFlow/FeatureCheckDataFlow.cs

```diff
@@ -337,7 +337,7 @@ static void And ()
 			}
```

Comment:
https://github.com/dotnet/runtime/pull/102679/files#diff-6283f7ce8ea2a4569a8265503cdd55671a790101ca3c0fd4abd6110c605e30eeL326-L337

This is like the ones I couldn't find an issue for.

Reply:
Made https://github.com/dotnet/runtime/issues/102830 to track unifying behavior on this.